documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-monitoring-streaming-metrics.adoc

```diff
@@ -18,6 +18,18 @@ The following table lists the streaming metrics that are available.
 |`long`
 |The total number of events that this connector has seen since last started or reset.
 
+|[[connectors-strm-metric-totalnumberofcreateeventsseen_{context}]]<<connectors-strm-metric-totalnumberofcreateeventsseen_{context}, `TotalNumberOfCreateEventsSeen`>>
+|`long`
+|The total number of create events that this connector has seen since last started or reset.
+
+|[[connectors-strm-metric-totalnumberofupdateeventsseen_{context}]]<<connectors-strm-metric-totalnumberofupdateeventsseen_{context}, `TotalNumberOfUpdateEventsSeen`>>
+|`long`
+|The total number of update events that this connector has seen since last started or reset.
+
+|[[connectors-strm-metric-totalnumberofdeleteeventsseen_{context}]]<<connectors-strm-metric-totalnumberofdeleteeventsseen_{context}, `TotalNumberOfDeleteEventsSeen`>>
+|`long`
+|The total number of delete events that this connector has seen since last started or reset.
```

Comment:
You means that delete event which including truncate event?
If yes, I will add truncate event in updateCommonEventMetrics method.

    private void updateCommonEventMetrics(Operation operation) {
        totalNumberOfEventsSeen.incrementAndGet();
        lastEventTimestamp.set(clock.currentTimeInMillis());
        if (operation != null) {
            switch (operation) {
                case CREATE:
                    totalNumberOfCreateEventsSeen.incrementAndGet();
                    break;
                case UPDATE:
                    totalNumberOfUpdateEventsSeen.incrementAndGet();
                    break;
                case DELETE:
                case TRUNCATE:
                    totalNumberOfDeleteEventsSeen.incrementAndGet();
                    break;
                default:
                    break;
            }
        }
    }

Reply:
Ah, I thought this would already be the case? If that's actually not true, i.e. the "delete" count only counts DELETE but not TRUNCATE events, let's keep it that way. We may add a separate "truncate" counter as a follow-up, if there's demand for it.

Reply:
Ok, just took a look myself, we only count delete events atm. So that's good then.

Reply:
Ok, keep delete event as the delete metric